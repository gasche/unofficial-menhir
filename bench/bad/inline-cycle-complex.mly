%start<int> a
%token INT
%%

a: b {}

%inline b: INT c INT {}
%inline c: d INT d {}
%inline d: b {}
