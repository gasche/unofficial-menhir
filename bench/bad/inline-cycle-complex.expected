File "inline-cycle-complex.mly", line 9, characters 8-9:
Error: there is a cycle in the definition of d.
